worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/regen/Regen_v1_15_R2.java

```diff
@@ -182,6 +184,14 @@ protected boolean initNewWorld() throws Exception {
             @Override
             public void doTick(BooleanSupplier booleansupplier) { //no ticking
             }
+
+            @Override
+            public BiomeBase a(int i, int k, int j) {
+                if (options.hasBiomeType()) {
+                    return IRegistry.BIOME.get(MinecraftKey.a(options.getBiomeType().getId()));
```

Comment:
Is it better worth having this cached rather than obtaining every time?

Reply:
For sure. I'll try to work that out later today if I find the time.

Reply:
Resolved in https://github.com/IntellectualSites/FastAsyncWorldEdit/pull/707/commits/27f16d97b31af01773deecb5760538cd71cefe54